markdown_lab/core/scraper.py

```
@@ -221,7 +100,11 @@
         """
         import time
         import tracemalloc
-        import psutil  # type: ignore
+        try:
+            import psutil  # type: ignore
```

Ruff (0.8.2)

`psutil` imported but unused; consider using `importlib.util.find_spec` to test for availability

(F401)